pytest_xvfb.py

```diff
@@ -132,6 +133,11 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     if config.getoption('--no-xvfb') or not xvfb_available():
         config.xvfb = None
+        if (sys.platform.startswith('linux') and 'DISPLAY' in os.environ and
+                not config.getoption('--no-xvfb')):
+                    print(
+                        'pytest-xvfb could not find Xvfb. '
+                        'You can install it to avoid windows from opening.')
```

Comment:
Please dedent this by 8 spaces - as per [pep8](https://www.python.org/dev/peps/pep-0008/) one of the possibilities with multi-line if's (the one I happen to prefer :laughing:) is:

```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```